src/style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  margin: 0;
+}
+
+#header {
+  width: 300px;
+  height: 100px;
+  margin-right: auto;
+  margin-left: auto;
+  background-color: blue;
+}
+
+#header:hover {
+  height: 200px;
+}
+
+.important {
+  width: 300px;
+  height: 100px;
+  background-color: red;
+  margin-right: auto;
+  margin-left: auto;
```

Comment:
you repeat it several times, try to not do this

Reply:
Can you write me what I need to try do not repeat exactly?

Reply:
done